### ISSUE TITLE
Internal server error on some urls

### ISSUE THREAD

**Issue by tzarebczan** [lbryio/lighthouse#155]
https://lighthouse.lbry.com/autocomplete?s=@swiss%20experiments

**Comment by tiger5226** [comment]
reproduced it locally

error: `TypeError: Cannot read property 'metadata' of null`

for 

```js
if (pResult._source.value && pResult._source.value.stream !== undefined) {
          var title = pResult._source.value.stream.metadata.title;
          var author = pResult._source.value.stream.metadata.author;
          if (title.indexOf(ctx.query.s.trim()) > -1 && title.indexOf('http') === -1) {
            cResults.push(title);
          }
          if (author.indexOf(ctx.query.s.trim()) > -1 && author.indexOf('http') === -1) {
            cResults.push(author);
          }
        }
```

specifically `var title = pResult._source.value.stream.metadata.title`.

**Comment by tiger5226** [comment]
So a `stream` is not undefined but it's null? 

@seanyesmunt Should I be checking for both null and undefined?

**Comment by tzarebczan** [comment]
Is it pulling from the json data in CQ? Ew. The structure changed with updates....it's just `source.value.title` now, but it should just pull from the CQ table directly instead.

**Comment by tiger5226** [comment]
Yes it is and yeah, agreed. Will be changing that. However, Chainquery is backwards compatible. So that should not be the problem. I made the changes in chainquery to not change the structure of that json column. It was a little extra work but it made sure the API ( SQL queries ) would not break. This is something else.

I am checking `undefined` but not null and it looks like the first check is checking for null.

**Comment by tiger5226** [comment]
chainquery https://github.com/lbryio/chainquery/blob/94a8ea83e14bd6815814d8f5c02ae8156dffc1d3/daemon/processing/metadata.go#L50-L85

**Comment by tzarebczan** [comment]
It's a channel, it won't even have a stream section. 

This is what I'm seeing on CQ in value as json. 
```{"Claim":{"claimType":"certificateType","stream":null}}
```

**Comment by tiger5226** [comment]
yeah, and I am checking `undefined` but its `null`.

 https://www.ajaymatharu.com/javascript-difference-between-undefined-and-null/

Based on the above I think I need to check for both null and undefined. Changing this now to see if its resolved.

**Comment by tiger5226** [comment]
fixed with 3a061df1a4ebd536a49d95d2dd446faad4345a9b
